src/templates/components/forms_common_rows.html.tt

```diff
@@ -44,6 +44,26 @@
         hint=hint || (no_hint ? '' : l('Kindly provide your complete address.<br>This will be used to authenticate your identity when you open a real account.')) %]
 [% END %]
 
+[% BLOCK address_validation_header %]
+    <span>[% l("Please refer to the") %]<strong>&nbsp[% l("characters that can be used") %]</strong></span>
```

Comment:
this line will not be possible for translators to translate properly, because they will see "Please refer to the" and "characters that can be used" as two separate sentences. it's better to use something like `[% l("Please refer to the [_1]characters that can be used", "<strong>") %]` if you want to avoid adding the tag in the sentence.